plugins/healthchecks/Makefile.inc

```diff
@@ -14,9 +14,5 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-if BUILD_HEALTHCHECK_PLUGIN
```

Comment:
unrelated, but general cleanup. we conditionalize compilation at the parent Makefile for everything else.